beacon-chain/rpc/beacon/assignments.go

```diff
@@ -134,42 +138,48 @@ func (bs *Server) ListValidatorAssignments(
 	}, nil
 }
 
-// NextEpochProposerList retrieves the validator assignments for a given epoch,
-// [This api is specially used for Orchestrator client]
-// optional validator indices or public keys may be included to filter validator assignments.
-func (bs *Server) NextEpochProposerList(
-	ctx context.Context, empty *ptypes.Empty) (*ethpb.ValidatorAssignments, error) {
-
-	curEpoch := helpers.SlotToEpoch(bs.GenesisTimeFetcher.CurrentSlot())
-	nextEpoch := curEpoch + 1
-
-	curEpochStartSlot, err := helpers.StartSlot(curEpoch)
-	if err != nil {
-		return nil, err
-	}
-
-	nextEpochStratSlot, err := helpers.StartSlot(nextEpoch)
+func (bs *Server) GetProposerListForEpoch(
```

Comment:
Why do we need this `GetProposerListForEpoch` or `NextEpochProposerList ` now? We do not call this api now.

Reply:
we want to fetch certain epoch, its not necessary `next`

Reply:
Yeah, And we also do not need `GetProposerListForEpoch` this method too. This method used for grpc api over http.

Reply:
It's now private for `bs - beacon server` and we'are using it inside `GetMinimalConsensusInfo`